src/api/controllers/TablesController.ts

```diff
@@ -26,11 +26,13 @@ class TablesController {
           isAuthorized = hasAuthorization(table.roles.read, req.user)
         }
         if (isAuthorized) {
-          return {
-            name: table.name,
-            verbose: table.verbose,
-            order: table.order || undefined,
-          }
+          const result: any = {}
+          Object.keys(table).forEach((key) => {
+            if (!['columns', 'relations'].includes(key)) {
+              result[key] = (table as any)[key]
```

Comment:
Please check this a little bit more to see if it's possible to remove the as any.